src/main/java/com/techeer/fashioncloud/FashioncloudApplication.java

```diff
@@ -4,16 +4,18 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
 @EnableScheduling
 @EnableCaching
 @EnableAspectJAutoProxy
+@EnableJpaAuditing
```

Comment:
```suggestion
@EnableScheduling
@EnableCaching
@EnableAspectJAutoProxy
@EnableJpaAuditing
```
추천 예시 : 설정 별 각각 만들기
@EnableJpaAuditing 경우

@EnableJpaAuditing
@Configuration
JpaAudititngConfig(){
}